### PR TITLE
Andy/schema import bug fix

### PR DESCRIPTION
**Pull request by andrew-wm-arthur**

### CODE REVIEW

File: bats/create-tables.bats [hunk 1]
```diff
@@ -3,120 +3,40 @@ load $BATS_TEST_DIRNAME/helper/common.bash
 
 setup() {
     setup_common
-}
```
Comment (andrew-wm-arthur) [inline bats/create-tables.bats:6]: all deleted tests were copied to `sql-create-table.bats

**Comment by timsehn** [comment]
Maybe a little summery of what you did to bats and why before I jump in. This is rather large :-)

File: bats/capital-letter-column-names.bats [hunk 2]
```diff
@@ -3,7 +3,22 @@ load $BATS_TEST_DIRNAME/helper/common.bash
...
+    Bbb VARCHAR(20),
+    Ccc VARCHAR(20),
+    PRIMARY KEY (Aaa)
+);
+SQL
+    cat <<DELIM > capital-letter-column-names.csv
+Aaa,Bbb,Ccc
+1,aaa,AAA
+2,bbb,BBB
+DELIM
```
Comment (timsehn) [inline bats/capital-letter-column-names.bats:19]: I like how you deleted these helper files and put them in here. Worried a little about the filesystem interaction not being tested but I think we get enough coverage still elsewhere.

File: bats/sql-create-tables.bats [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+    skip "Current SQL parser requires backticks around keywords, not just reserved words"
+    [ "$status" -eq 0 ]
+    run dolt schema show
+    [ "$status" -eq 0 ]
+    [[ "$output" =~ "comment" ]] || false
+    [[ "$output" =~ "date" ]] || false
+}
+
+
+@test "create two table with the same name" {
```
Comment (timsehn) [inline bats/sql-create-tables.bats:310]: tables

File: bats/schema-import.bats [hunk 4]
```diff
@@ -174,7 +184,6 @@ teardown() {
 
 @test "schema import of two tables" {
     dolt schema import -c --pks=pk test1 `batshelper 1pksupportedtypes.csv`
-    skip "Guaranteed tag collision right now"
```
Comment (timsehn) [inline bats/schema-import.bats:177]: These are the droids I'm looking for.

**Comment by timsehn** [comment]
> Maybe a little summery of what you did to bats and why before I jump in. This is rather large :-)

Nevermind. I like how you moved the craete-table tests into their own file.

File: bats/capital-letter-column-names.bats (hunk 2 above)
Comment (andrew-wm-arthur) [reply bats/capital-letter-column-names.bats:19]: We use `stdin` with `<<SQL ... SQL` a lot, but all of these changes are catted to a real file before being opened by Dolt

**Pull request merged**